tools/development/gstPrototxt.py

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env python3
+##
```

Comment:
for scripts, SPDX can be at max at line 2 (https://lore.kernel.org/patchwork/patch/883424/)

Reply:
Applied.